### PR TITLE
Hw01 participant review

### PR DESCRIPTION
**Pull request by Han-Lee93**
@gregrousis
@diegodulanto

### CODE REVIEW

File: HW01/readme.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+| Comfortable    | :thumbsup:    |
+| Adequate       | :ok_hand:     |
+| Needs work     | :thumbsdown:  |
+
+- [:thumbsdown:] Using Github
+- [:ok_hand:] Programming
+- [:thumbsup:] Using R
+- [:ok_hand:] Data organization/management using R
+- [:thumbsdown:] Comfortability with cloud-based platforms
+
```
Comment (diegodulanto) [inline HW01/readme.md:35]: Wonderful work! The final result looks amazing!

File: HW01/HW1 - Cars dataset Finalized.Rmd [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Details regarding dataset cars
+```{r Cars}
+# Shows dataset
+cars
+# Summary data of cars dataset
+summary(cars)
+# Correlation of distance and speed
+cor(cars$speed, cars$dist)
+```
+
```
Comment (diegodulanto) [inline HW01/HW1 - Cars dataset Finalized.Rmd]: Nice work!

File: HW01/HW1 - Cars dataset Finalized.Rmd [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+   add = "reg.line",  # Add regressin line
+   add.params = list(color = "blue", fill = "lightgray"), # Customize reg. line
+   conf.int = TRUE, # Add confidence interval
+   title = "Correlation between speed and distance" # Add title
+   )
+# Add correlation coefficient to plot
+sp + stat_cor(method = "pearson", label.x = 3, label.y = 30)
+#> `geom_smooth()` using formula 'y ~ x'# Plot means of car dataset
+```
+
```
Comment (diegodulanto) [inline HW01/HW1 - Cars dataset Finalized.Rmd]: Great job!

**Comment by diegodulanto** [comment]
You did an awesome job!

File: HW01/readme.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Han Lee's Data Programming Repository
+This repository contains files and folders related to the *Data Programming class (Spring 21)*. This includes HW assignments, experimental code, and storage for items such as images.
+
+## Introduction - About Me
+
+I am currently a 4th year graduate student at the University of South Florida. I am from Ohio and I got my B.S. at the Ohio State University.
+
+<img src="https://github.com/usf-progdata/hw-Han-Lee93/blob/HW01/Images/usf.png" width="150" height="150" align="left">  <img src="https://github.com/usf-progdata/hw-Han-Lee93/blob/HW01/Images/osu.png" width="150" height="150" align="right">
+<br clear="right"/>
```
Comment (gregrousis) [inline HW01/readme.md:9]: Looks really nice! Very clean and visually appealing.

File: HW01/readme.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+|----------------|---------------|
+| Comfortable    | :thumbsup:    |
+| Adequate       | :ok_hand:     |
+| Needs work     | :thumbsdown:  |
+
+- [:thumbsdown:] Using Github
+- [:ok_hand:] Programming
+- [:thumbsup:] Using R
+- [:ok_hand:] Data organization/management using R
+- [:thumbsdown:] Comfortability with cloud-based platforms
```
Comment (gregrousis) [inline HW01/readme.md:34]: I like this a lot!

File: HW01/HW1---Cars-dataset-Finalized.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+```r
+# Correlation of distance and speed
+cor(cars$speed, cars$dist)
+```
+
+```
+## [1] 0.8068949
+```
+
+## Plots
```
Comment (gregrousis) [inline HW01/HW1---Cars-dataset-Finalized.md:89]: This graph isn't showing up in the knitted doc. Is it supposed to be an attached image or an r plot?

Comment (gregrousis) [reply HW01/HW1---Cars-dataset-Finalized.md:89]: Might be a bug in .md files? I had to display the rich diff below to get the image to display, but it's in a separate github file, I think. Other than that, nice job!!

**Pull request merged**